sdk/confluent/Azure.ResourceManager.Confluent/samples/Generated/Samples/Sample_ConfluentOrganizationResource.cs

```diff
@@ -268,7 +268,7 @@ public async Task GetRegions_OrganizationListRegions()
         public async Task CreateApiKey_OrganizationCreateAPIKey()
         {
             // Generated from example definition: specification/confluent/resource-manager/Microsoft.Confluent/stable/2024-02-13/examples/Organization_CreateClusterAPIKey.json
-            // this example is just showing the usage of "Organization_CreateApiKey" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "Organization_CreateAPIKey" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it should be `API`: https://github.com/Azure/azure-rest-api-specs/blob/95aa73b8831deb2940ac7067b8f296cd189a5fe1/specification/confluent/resource-manager/Microsoft.Confluent/stable/2024-07-01/confluent.json#L1564